cfme/tests/test_rest.py

```diff
@@ -1119,3 +1112,37 @@ def test_find_created_events(self, appliance, vm_obj, gen_events, provider, soft
                 evt_col.get(id=found_evts[-1].id)
             except (IndexError, ValueError):
                 soft_assert(False, "Couldn't get event {} for vm {}".format(evt, vm_name))
+
+
+@pytest.mark.tier(3)
+@pytest.mark.parametrize("interval", ["hourly", "daily"])
+@pytest.mark.parametrize("resource_type", ["VmOrTemplate", "Service"])
```

Comment:
Optional, the double parametrization can be done in a single statement:

`@pytest.mark.parametrize(("param1, param2"), [("value11", "value12"), ("value21", "value22")]`)

Reply:
I think parametrizing this way will not give me the combinations that I require.
I need 4 combinations - Service-hourly, Service-weekly, VmOrTemplate-hourly and VmOrTemplate-weekly
Using 2 parametrize statements helps in getting the combination.